src/sentry/integrations/github/client.py

```diff
@@ -194,44 +194,43 @@ def _populate_repositories(self, gh_org: str, cache_seconds: int) -> List[Dict[s
 
         return repositories
 
+    def _populate_trees_process_error(self, error: ApiError, extra: Dict[str, str]) -> None:
```

Comment:
I just pulled this function from within `_populate_trees` so the function is easier to read.

<img width="1284" alt="image" src="https://user-images.githubusercontent.com/44410/221661886-6c1de258-203d-4998-9a7c-22192e861009.png">